registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -4040,6 +4040,328 @@
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "optimism",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "fraxtal",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "blast",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "kava",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "fantom",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "arbitrum",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "immutable",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "linea",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "STABLE",
+          "name": "STABLE",
+          "axelarChainId": "scroll",
+          "tokenAddress": "0x5120DAf6f323E03b70EFa289f1EDF0e5019d5436",
+          "tokenManager": "0xe7c4dD18E24bdEa9886a96D780F3Fa2B7252Cc85",
+          "tokenManagerType": "mintBurn"
+        }
+      ]
     }
   }
-}
```

Comment:
This looks good, but it's a tiny fix here. linting failed; it seemed you added extra } or were missing.

Kindly take a look so we can review and merge asap. Thank you for the effort so far. @DerDepp